src/scripts/crowdin/getCrowdinContributors.ts

```diff
@@ -0,0 +1,17 @@
+import getAndSaveDirectories from "./source-files/fetchAndSaveDirectories"
+import getDirectoryIds from "./utils"
+import { getTranslatedMarkdownPaths } from "../markdownChecker"
+import fetchAndSaveFileIds from "./source-files/fetchAndSaveFileIds"
+import getTranslationCostsReport from "./reports/getTranslationCostsReport"
+
+async function main() {
+  await getAndSaveDirectories()
+  const directoryIds = getDirectoryIds()
+  await fetchAndSaveFileIds(directoryIds)
```

Comment:
Inconsistent function naming here, bit of a smell but don't think it should block.